lib/cucumber/support_code/step_definition.js

```diff
@@ -1,3 +1,4 @@
+var realTime = require('../util/real_time');
```

Comment:
Instead of requiring this directly, please set `Cucumber.Util.RealTime` in `lib/cucumber/util.js`